transformers/events/DOCUMENTATION.md

```diff
@@ -2,116 +2,206 @@
+      deployed = 8928180
+```
+* The address is the signature you searched for, as a string.
+* The abi can be found on the `contracts` tab in Etherscan (scroll down).
+You'll need the entire thing as a string, so it's best to use single quotes
+like above.
+* The deployed field is the block number the contract was deployed on. That
+can be found on the Contract's page, by looking for the Contract Creator
+and clicking it's transaction hash. That will take you to the block it was deployed on.
+
+### Create the Transformer
+
+To create a custom transformer you will need to create a new `Transformer`
+struct with a `ToModels` method on it which converts a `core.EventLog` object
+(the raw, untransformed data) to an `event.InsertionModel` (the domain object).
+
+Note that for this step you probably do _not_ need to create the database
+migration. This is because the only thing you'll be saving are addresses, which
+go into the already existing address table. If you do need to create the
+migration, you'll know, because the tests won't pass. The directions here assume
+you do not, yet.
+
+1. Search for the contract on etherscan using it's signature. 
```

Comment:
I think this `signature` should be `address` here as well.

Reply:
👍